scripts/languageserver/parse.jl

```diff
@@ -0,0 +1,243 @@
+type VarInfo
+    t
```

Comment:
Could you either use self-explanatory names, or add a comment in a case like this that says what this field stores?